DocSum/docker_image_build/build.yaml

```diff
@@ -11,18 +11,57 @@ services:
       context: ../
       dockerfile: ./Dockerfile
     image: ${REGISTRY:-opea}/docsum:${TAG:-latest}
+
   docsum-ui:
     build:
-      context: ../ui
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+      context: ../ui/
       dockerfile: ./docker/Dockerfile
     extends: docsum
     image: ${REGISTRY:-opea}/docsum-ui:${TAG:-latest}
-  docsum-react-ui:
```

Comment:
Why did you remove this image?  
And I didn't see the new image `docsum-gradio-ui`, if the image not in this file, it won't be released on docker hub.